production.Dockerfile

```diff
@@ -179,4 +179,8 @@ EXPOSE 8000
 # Expose the port from which we serve OpenMetrics data
 EXPOSE 8001
 
+# Use tini as the entrypoint such that signals are propagated to subprocesses
+RUN apk add --no-cache tini==0.19.0-r0
```

Comment:
Nit: not sure we should pin till r0 as this will likely break once the package will be removed from the repo